examples/json_obj/json_obj.go

```diff
@@ -78,8 +82,8 @@ func main() {
 	fmt.Println("Memory used by obj:", res)
 
 	res, err = rejson.JSONGet(conn, "obj", ".",
-		&rejson.JSONGetOptionIndent{"\t"}, &rejson.JSONGetOptionNewLine{"\n"},
-		&rejson.JSONGetOptionSpace{" "}, &rejson.JSONGetOptionNoEscape{})
+		rejson.NewJSONGetOptionIndent("\t"), rejson.NewJSONGetOptionNewLine("\n"),
```

Comment:
Also you might want to pull first before submitting this as I am merging https://github.com/nitishm/go-rejson/pull/22 before I merge these changes.